### DISCUSSION TITLE
AWS - Admin Console Confirmation Email Not Sent/Received

### DISCUSSION THREAD

**Discussion by fagerburg** [mozilla/hubs#3796] [Hubs Cloud Questions] [answered]
I followed the documentation for setting up Hubs Cloud personal, when i access the domain, it prompts me to sign in to the admin console. I receive emails from AWS SES directly but I don't receive them from hubs so I can't even sign in to my deployed instance. I've verified both company internal and external emails. 

- Is there a way to get into the admin console without verifying email? Or some way to troubleshoot hubs emails? 

There is a topic in the troubleshooting area but it assumes that you already have access to the admin console which i don't have.

**Comment by BonzoMartini** [comment]
I have the same problem. Hubs Admin decided to log me out and now I can't log in. I don't know what steps there are to regain access to Hubs Admin if you don't have it. It doesn't seem possible to change the admin address of the Stack and update it. Don't know what to try - pretty much anything is a guess rn. Maybe its possible to restore from backup??

**Reply by xaddison** [reply]
I am subscribed because we have the same issue.  We had been getting emails for a month fine and they stopped. But they stopped in away that the configuration and whatever MacGyver-ing I did to make it work thru MailGun is gone now.

**Reply by robinkwilson** [reply]
@xaddison You can go back to the aws defaults with mail sending if you delete the MailGun settings in the Parameter Store: I helped someone through this process, there were 4 email settings that you had to delete: https://github.com/mozilla/hubs/discussions/3657

You can make a new discussion thread and I'll write more detailed instructions of how to delete the custom mail setup.

**Comment by robinkwilson** [comment, marked as answer]
Unfortunately you'll need to verify your account via email with hubs before you can get into the admin panel. 

So quick steps to try:
- Go to AWS management console > SES (simple email settings) > check that you verified your email address in the N. Virginia region with AWS (otherwise you will not get mail)
- Also check your junk mailbox for the link

**Reply by robinkwilson** [reply]
Marking this as answered for now, please _unmark_ if you have more info/questions on this!

Also note. All AWS emails go through N. Virginia region in AWS (regardless of deploy region, always N. Virginia)

**Comment by MyVersep** [comment]
Helli I have the same case, anyone a the issue ?
